src/main/java/uk/gov/gchq/gaffer/doc/operation/spark/GetJavaRDDOfElementsExample.java

```diff
@@ -95,10 +102,37 @@ public void getJavaRddOfElements(final JavaSparkContext sc, final Graph graph) t
         for (final Element e : elements) {
             log(e.toString());
         }
-        log("```");
+        log("```\n");
         ROOT_LOGGER.setLevel(Level.OFF);
     }
 
+    public void getJavaRddOfElementsWithHadoopConf(final JavaSparkContext sc, final Graph graph) throws OperationException, IOException {
+        ROOT_LOGGER.setLevel(Level.INFO);
+        log("#### get Java RDD of elements using Hadoop Configurations");
+        final Configuration conf = new Configuration();
+        conf.set("AN_OPTION", "A_VALUE");
+        final ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        conf.write(new DataOutputStream(baos));
+        final String encodedConf = Base64.encodeBase64String(baos.toByteArray());
```

Comment:
I think the configuration should be converted to a string using:

    final String configurationString = AbstractGetRDDHandler.convertConfigurationToString(configuration);

This is simpler for the user and guarantees that it will be converted in the correct way.

Reply:
I have created a new issue to update this when RC3 is released.